clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/DataRecoveryTask.java

```diff
@@ -19,6 +19,10 @@ public void run() {
     taskResult = new TaskResult(command.getResult());
   }
 
+  public boolean requiresSentinelRun() {
```

Comment:
Can you add some more details what this means and why we need it?

Reply:
Thanks Nisarg for the review! The change description part probably has more details and explains why. Do you mean to move some of them here as comments?

Reply:
Yeah, as javadoc or comments

Reply:
Sure, will add the comments and rename function to make it less confusing as we discussed offline.